### PR TITLE
SAK-49793 Polls implement the option to remove/cleanup during site import

### PR DESCRIPTION
**Pull request by maurercw**
…es by deleting any polls in the destination site

### CODE REVIEW

File: polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java [hunk 1]
```diff
@@ -546,7 +546,18 @@ public String[] myToolIds()
...
+			if (cleanup) {
+                try {
+                    // Find and delete any polls from the destination site
+                    List<Poll> toPolls = findAllPolls(toContext);
+                    for (Poll poll : toPolls) {
+                        deletePoll(poll);
+                    }
+                } catch(Exception e) {
+                    log.error(e.getMessage(), e);
+                }
```
Comment (bgarciaentornos) [inline polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java:559]: Please fix spacing issues, thanks!

Comment (ottenhoff) [reply polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java:559]: what's the spacing issue? tabs?

Comment (adrianfish) [reply polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java:559]: It's just polls, man, just polls ... spacing all over the shop

Comment (ottenhoff) [reply polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java]: i see spaces vs tabs. i went in and edited

**Review by jesusmmp — APPROVED** [review]

File: polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java (hunk 1 above)
Comment (bgarciaentornos) [reply polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java]: Thanks, it looks fine now!

**Pull request merged**